.storybook/components/DesignTokens/Tier2/TypographyUsage.module.css

```diff
@@ -5,137 +5,114 @@
+ *  Use typography tokens to display the type ramp
  */
 
+/* TODO-AH: these should actually use the relevant tier 2 tokens. the tier 3 ones need to be on their own page */
+
 .typography-usage--headline-lg {
-  @mixin eds-typography-preset-001-bold;
+  font: var(--eds-typography-preset-001-bold);
 }
 
 .typography-usage--headline-md {
-  @mixin eds-typography-preset-002-bold;
+  font: var(--eds-typography-preset-002-bold);
 }
 
 .typography-usage--headline-sm {
-  @mixin eds-typography-preset-003-bold;
+  font: var(--eds-typography-preset-003-bold);
 }
 
 .typography-usage--title-md {
-  @mixin eds-typography-preset-004-bold;
+  font: var(--eds-typography-preset-004-bold);
 }
 
 .typography-usage--title-sm {
-  @mixin eds-typography-preset-005-bold;
+  font: var(--eds-typography-preset-005-bold);
 }
 
 .typography-usage--title-xs {
-  @mixin eds-typography-preset-007-bold;
+  font: var(--eds-typography-preset-007-bold);
 }
 
 .typography-usage--label-lg-subtle {
-  @mixin eds-typography-preset-005;
+  font: var(--eds-typography-preset-005);
 }
 
 .typography-usage--label-md-subtle {
-  @mixin eds-typography-preset-007;
+  font: var(--eds-typography-preset-007);
 }
 
 .typography-usage--label-md {
-  @mixin eds-typography-preset-007-bold;
+  font: var(--eds-typography-preset-007-bold);
 }
 
 .typography-usage--label-sm {
-  @mixin eds-typography-preset-009-bold;
+  font: var(--eds-typography-preset-009-bold);
 }
 
 .typography-usage--body-lg {
-  @mixin eds-typography-preset-004;
+  font: var(--eds-typography-preset-004-light);
 }
 
 .typography-usage--body-md {
-  @mixin eds-typography-preset-005-light;
+  font: var(--eds-typography-preset-005-light);
 }
 
 .typography-usage--body-sm {
-  @mixin eds-typography-preset-006;
+  font: var(--eds-typography-preset-006-light);
 }
 
 .typography-usage--body-xs {
-  @mixin eds-typography-preset-008;
+  font: var(--eds-typography-preset-008-light);
 }
 
 .typography-usage--caption-lg {
-  @mixin eds-typography-preset-006;
+  font: var(--eds-typography-preset-006);
 }
 
 .typography-usage--caption-md {
-  @mixin eds-typography-preset-008;
+  font: var(--eds-typography-preset-008);
 }
 
 .typography-usage--caption-sm {
-  @mixin eds-typography-preset-010;
+  font: var(--eds-typography-preset-010);
 }
 
 .typography-usage--button-label {
-  @mixin eds-typography-preset-006-bold;
+  font: var(--eds-typography-preset-006-bold);
 }
 
 .typography-usage--form-label {
-  @mixin eds-typography-preset-006-bold;
+  font: var(--eds-typography-preset-006-bold);
 }
 
 .typography-usage--form-input {
-  @mixin eds-typography-preset-005;
+  font: var(--eds-typography-preset-005);
 }
 
 .typography-usage--overline {
-  @mixin eds-typography-preset-011-bold;
+  font: var(--eds-typography-preset-011);
+  text-transform: uppercase;
 }
 
 .typography-usage--overline-sm {
-  @mixin eds-typography-preset-012-bold;
+  font: var(--eds-typography-preset-012);
+  text-transform: uppercase;
```

Comment:
Are we removing the larger weight treatment on the overline tokens?

Reply:
Actually, food for thought, perhaps overline can be a `Text` component variant only and not a token itself since it also has `text-transform`

Reply:
This is the confusing thing... as I was working thru things, i was just applying the values in our designs, and it says we should have had this?

![Screenshot 2023-06-28 at 18 13 10](https://github.com/chanzuckerberg/edu-design-system/assets/3056447/c94c8c7f-5f24-4aa0-8c3d-64b1c47c893e)

I don't know if this had been out of sync for a while, or was a recent change (pls add some kinda diffing to Figma)

Reply:
Think it's been out of sync for a while and never got noticed since this token prob has low usage... could consider even removing these

Reply:
@jinlee93 Ohh, yeah, all usages could be run thru `Text` for clarity and simplicity. This would give us a nice way to avoid exposing this `text-transform` thing.

Then we can greatly simplify these typography token pages and probably remove the css modules for those